session-replay/Sources/DatadogSessionReplay/Recorder/SnapshotBuilder/ViewTreeSnapshot.swift

```diff
@@ -176,14 +176,14 @@ internal struct InvisibleElement: NodeSemantics {
 /// A semantics of an UI element that is of `UIView` type. This semantics mean that the element has visual appearance in SR, but
 /// it will only utilize its base `UIView` attributes. The full identity of the node will remain ambiguous if not overwritten with `SpecificElement`.
 internal struct AmbiguousElement: NodeSemantics {
-    static let importance: Int = 1
+    static let importance: Int = 0
     let wireframesBuilder: NodeWireframesBuilder?
 }
```

Comment:
This makes `InvisibleElement.importance` and `AmbiguousElement.importance` equal (`0`).

**Rationale:** 
* The `AmbiguousElement` semantics is recorded by `UIViewRecorder`, which doesn't know the specific `UIView` subclass (that one is recorded later by specialised node recorders). 
* The `InvisibleElement` semantics can be recognised by any node recorder. In `UIViewRecorder` it is determined by `view.isHidden` and `view.alpha`, whereas in specialised recorders this is paired with high level aspects of specialised view, e.g. empty text in `UILabel` (which makes the node effectively invisible, even if `.alpha > 0` and `.isHidden == false`).

Because `UIViewRecorder` is queried first, the importance of `InvisibleElement` cannot be lower than importance of `AmbiguousElement`. Otherwise, specialised recorders wouldn't be able to override `AmbiguousElement` with `InvisibleElement`.

Reply:
💡 Perhaps we could go with no `importance` and record an array of semantics that will be further resolved in `Processor`:
```
struct Node {
   /// Different semantics recorded for this node.
   let semantics: [NodeSemantics]
}
```
I'd leave this idea for later - might come handy if we find `.importance` limited for more specialised views and complex hierarchies (e.g. `NodeRecorders` clashing with each other).

Reply:
I think it is fine to have same importance, it's like priority value 👍

Reply:
Or maybe the `InvisibleElement.importance` can be `-1`?